Jenkinsfile

```diff
@@ -126,8 +126,8 @@ DFLAGS=-I%@P%/../imports -L-L%@P%/../libs -L--export-dynamic -L--export-dynamic
         '''
         archiveArtifacts artifacts: 'distribution.tar', onlyIfSuccessful: true
     }
-}
 
-stage("Test downstream projects") {
-    build job: 'dlangci-downstream'
 }
+
+// executed from workspace root
+load('dlangci/ProjectsJenkinsfile')
```

Comment:
So far I've only manually build triggered builds, but the git/SCM plugin has a way to do that.
http://stackoverflow.com/a/9791873